code/src/index.css

```diff
@@ -5,9 +5,103 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #fff;
+  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+form {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  width: 50%;
+  padding: 20px;
+  background-image: linear-gradient(160deg, #4d3ccd 0%, #3a26c5 100%);
+  border-radius: 20px;
+  color: #fff;
+  -webkit-box-shadow: 2px 4px 17px -1px rgba(0,0,0,0.3);
+  -moz-box-shadow: 2px 4px 17px -1px rgba(0,0,0,0.3);
+  box-shadow: 2px 4px 17px -1px rgba(0,0,0,0.3);
+  
+}
+
+.wrapper-form{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 20px;
+  margin-top: 20px;
+}
+
+label {
+  padding: 30px;
+  
+}
+ 
+.name-question label {
+  display: flex;
+  flex-direction: column;
+}
+
+
+input:valid {
+  border-color:#9bd61c;
+}
+
+input:invalid {
+  border-color: red;
+}
+
+input[type=text], select {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  font-size: 18px; 
+}
+
+button {
+  display: flex;
+  justify-content: center;
+  width: 20%;
+  margin-top: 40px;
+  background-color: #fdc632; 
+  border: none;
+  border-radius: 10px;
+  color: #000;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 20px;
+  cursor: pointer; 
+}
+
+
+.summary-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  padding: 20px;
+}
+
+.kitten-gif {
+  width: 90%;
+  max-height: 380px;
+}
+
+.paws {
+  width: 95%;
+  margin-left: 40px;  
+}
+
```

Comment:
You have 2 class names in your js files that are not mentioned in your .css file. It may be a good idea to remove the unused class names from your code: app-container, summary-container.
(Well, you may need "summary-container" to style kitten.gif properly but, right now, it is not used 😏)